discretize/TensorMesh.py

```diff
@@ -159,6 +159,24 @@ def gridN(self):
         """Nodal grid."""
         return self._getTensorGrid('N')
 
+    @property
+    def h_gridded(self):
+        """
+        Returns an (nC, dim) numpy array with the widths of all cells in order
+        """
+
+        if self.dim == 1:
+            return self.h
```

Comment:
I believe this is len `(nC,)` rather than what is stated in the doc string. Probably what is wanted, but a bit confusing. Could use the `mkvc(self.h, 1)` I think? to add a blank axis?